promql/parser/printer.go

```diff
@@ -91,13 +91,19 @@ func (node *AggregateExpr) getAggOpStr() string {
 }
 
 func joinLabels(ss []string) string {
+	labelList := ""
 	for i, s := range ss {
+		if i > 0 {
+			labelList += ", "
```

Comment:
If this is called in performance-sensitive circumstances then use a buffer or builder.